www/source/blog/2017-10-20-node-scaffolding-scripts.html.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Node Scaffolding now supports pre and post build scripts!"
+date: 2017-10-20
+author: Nell Shamrell-Harrington
+tags: Node, scaffolding
+category: community
+classes: body-article
+---
+
+We have just released a new version of the Node scaffolding that supports scripts defined in package.json!
+
+It is common for Node applications to include scripts in their package.json files like so:
+
+**package.json**
+
+```
+"scripts": {
+	"start": "react-scripts start",
+	"build": "react-scripts build",
+	"test": "react-scripts test --env=jsdom",
+	"eject": "react-scripts eject"
+}
+```
+
+The [Habitat Node Scaffolding](https://github.com/habitat-sh/core-plans/tree/master/scaffolding-node) now supports these scripts out of the box.
+
+Let's take it for a test run:
+
+One of my favorite sample React apps is the [Pokedex](https://alik0211.ru/pokedex/) by Ali Gasymov.
+
+Let's habiterize this application and deploy it in a Docker container!
```

Comment:
I don't have any opinion at all on this, but I've seen `Habitize` used recently, but we should probably be consistent. (Minimally, though, this seems like a capital-H-type situation.) 😄